src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static final int ARRAY_SIZE = 10;
+
+    public static void main(String[] args) {
+        Figure[] figuresArray = new Figure[ARRAY_SIZE];
+        for (int i = 0; i < figuresArray.length / 2; i++) {
+            figuresArray[i] = new FigureSupplier().getRandomFigure();
+            figuresArray[i].draw();
+        }
+        for (int k = figuresArray.length / 2; k < figuresArray.length; k++) {
+            figuresArray[k] = new FigureSupplier().getDefaultFigure();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#you-should-create-several-random-figures-so-you-will-use-a-loop-please-dont-create-a-new-figuresupplier-inside-the-loop): You should create several random Figures, so you will use a loop. Please don’t create a new FigureSupplier() inside the loop.